ehcache-impl/src/main/java/org/ehcache/impl/internal/loaderwriter/writebehind/NonBatchingLocalHeapWriteBehindQueue.java

```diff
@@ -62,12 +67,18 @@ protected SingleOperation<K, V> getOperation(K key) {
   }
 
   @Override
+  @SuppressWarnings("unchecked")
   protected void addOperation(final SingleOperation<K, V> operation) {
     latest.put(operation.getKey(), operation);
 
     submit(() -> {
       try {
         operation.performOperation(cacheLoaderWriter);
+      }  catch (BulkCacheWritingException e) {
```

Comment:
This operation only does one write, so why are we special casing BulkCacheWritingException?